core/src/stages/move_relative.cpp

```diff
@@ -195,10 +195,18 @@ bool MoveRelative::compute(const InterfaceState& state, planning_scene::Planning
 
 	robot_trajectory::RobotTrajectoryPtr robot_trajectory;
 	bool success = false;
+	std::string error_message = "";
 
 	if (getJointStateFromOffset(direction, dir, jmg, scene->getCurrentStateNonConst())) {
 		// plan to joint-space target
-		success = planner_->plan(state.scene(), scene, jmg, timeout, robot_trajectory, path_constraints);
+		const auto planner_solution_maybe =
+		    planner_->plan(state.scene(), scene, jmg, timeout, robot_trajectory, path_constraints);
+		if (planner_solution_maybe.has_value()) {
+			success = planner_solution_maybe.value();
+		}
+		if (!success) {
+			error_message = planner_solution_maybe.error();
```

Comment:
Does this work? Doesn't the `.error()` operator only return a value if `.has_value == true`? I guess this section would be simpler if plan() doesn't return a boolean

Reply:
I have modified the plan function to return a MoveItErrorCode so now I check for the error type